lib/Screens/Loading.dart

```diff
@@ -15,7 +15,7 @@ class _LoadingState extends State<Loading> {
       home: SafeArea(
         child: Container(
           child: Center(
-              child: SpinKitPouringHourglass(
+              child: SpinKitRotatingCircle(
```

Comment:
```suggestion
              child: SpinKitPouringHourglass(
```

@letsintegreat, why was this changed? If there are no null safety issues then please revert this back. You can commit directly from here.